shortstories/models.py

```diff
@@ -13,3 +13,6 @@ class ShortStory(models.Model):
     body = models.TextField(default= "No story written by the author")
     status = models.CharField(max_length = 1, choices = PUBLICATION_STATUS)
     publication_date = models.DateField()
+
+    def Snippet(self):
```

Comment:
Please read the official python [guide](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names), function names should start with lower case, classes will start with upper case. Also

- [ ] Why do you need parentheses around your return statement
- [ ] Could you check if body is None `return ""`

Reply:
if body is None, it should return default or just " " ?

Reply:
just blank.